javascript/javascript3/week2/CurrencyCalculator/script.js

```diff
@@ -0,0 +1,78 @@
+const inputConvertFrom = document.querySelector('#from-input'),
+      inputConvertTo = document.querySelector('#to-input'),
+      selectFrom = document.querySelector('#from-select'),
+      selectTo = document.querySelector('#to-select');
+
+
+
+fetch('https://open.er-api.com/v6/latest/USD')
+    .then(response => response.json())
+    .then((data) => {
+        console.log(data);
+        for (const key in data.rates) {
+            const convertFromCurrency = document.createElement('option');
+            convertFromCurrency.innerHTML = key;
+            selectFrom.appendChild(convertFromCurrency);
+            if (key.includes('EUR')) {
+                convertFromCurrency.setAttribute('selected', true);
+            }
+            
+
+            const convertToCurrency = document.createElement('option');
+            convertToCurrency.innerHTML = key;
+            selectTo.appendChild(convertToCurrency);
+            if (key.includes('DKK')) {
+                convertToCurrency.setAttribute('selected', true);
+            }
+        }
+
+        function currencyCalculator() {
+            let rateFrom,
+                rateTo;
+            for (const key in data.rates) {
+                if (selectFrom.value === key) {
+                    rateFrom = data.rates[key];
+                } else if (selectTo.value === key) {
+                    rateTo = data.rates[key];
+                } 
+
+                inputConvertTo.value = ((+inputConvertFrom.value / rateFrom) * rateTo).toFixed(2);
+                
+            if (inputConvertFrom.value === '') {
```

Comment:
In both this and the next funcion you could do this check at the begining and it it is true exit the funcion with `return` stetement, like that: 
```
function currencyCalculator() {
    if (inputConvertFrom.value === '') {
        inputConvertTo.value = '';
        return;
     }

// rest of the code, the for loop etc... 

}
 ```
That way you handle the edge case first and you are not looping through the object when input is empty

Reply:
Thanks for review and tips) I have made some changes